### ISSUE TITLE
Pre-trained models - low accuracy?

### ISSUE THREAD

**Issue by ameya98** [huyvnphan/PyTorch-CIFAR10#4]
When loading the pretrained ResNets, I am getting very low training set accuracy (around 22% on a subset of 1000 images).

This is a sample from my code:
```python
model = cifar10_models.resnet50(pretrained=True)

mean = np.array([0.4914, 0.4822, 0.4465])
std = np.array([0.2023, 0.1994, 0.2010])

self.normalize = transforms.Normalize(mean=mean, std=std)

self.train_loader = DataLoader(
            datasets.ImageFolder(training_dir, transform=transforms.Compose([
                transforms.RandomCrop(32, padding=4),
                transforms.RandomHorizontalFlip(),
                transforms.ToTensor(),
                self.normalize,
            ])),
            batch_size=1, shuffle=True,
            num_workers=0, pin_memory=True)
       
```
Can you please confirm that these weights are the right ones?

**Comment by ameya98** [comment]
The answer is to call `model.eval()` after loading the weights! This will tell the dropout and batch-normalization layers that we are in evaluation, and not training mode. This fixed my issue!
